docs/src/api/class-browsercontext.md

```diff
@@ -1102,7 +1102,6 @@ A glob pattern, regular expression or predicate to match the request URL. Only r
 
 ## method: BrowserContext.serviceWorkers
 * since: v1.11
-* langs: js, python
```

Comment:
This will be dropped from the changeset once https://github.com/microsoft/playwright/pull/15635 lands